modules/olm-building-operator-catalog-image.adoc

```diff
@@ -43,6 +43,23 @@ For example, for the `podman` CLI:
 ----
 $ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
 ----
+* If you are working with catalogs from private namespaces on
+link:https://quay.io[quay.io] that your account has access to, set the
+`AUTH_TOKEN` environment variable for use with the `--auth-token` flag by making
+a request against the login API using your link:https://quay.io[quay.io]
+credentials:
```

Comment:
Lots packed into one sentence, wonder if it would make sense still after splitting it into parts. Is the main point here that they must set the env variable or that they must hit the login API?

If the former, maybe something like:

> If you are working with catalogs that your account has access to from private namespaces on quay.io , you must set the `AUTH_TOKEN` environment variable for use with the -`-auth-token` flag. This is done by making a request against the login API using your quay.io credentials: 

If the latter, maybe something like:

> If you are working with catalogs that your account has access to from private namespaces on quay.io , you must make a request against the login API using your quay.io credentials. Set the `AUTH_TOKEN` environment variable for use with the -`-auth-token` flag: